rune/libenclave/internal/runtime/pal/api_linux_v1.go

```diff
@@ -8,15 +8,19 @@ static int palGetVersion(void *sym)
 	return ((int (*)(void))sym)();
 }
 
-static int palInitV1(void *sym, const char *args, const char *log_level)
+static int palInitV1(void *sym, const char *args, const char *log_level, long addr, int fd)
```

Comment:
All new codes should be placed to api_linux_v3.go. v1 is already freeze.

Reply:
ok Init has been added into V3.
I will delete it in v1.  This is my neglience